### PR TITLE
Add Anaconda CE manifest and secret

### PR DESCRIPTION
**Pull request by Xaenalt**

### CODE REVIEW

File: opendatahub.yaml [hunk 1]
```diff
@@ -41,6 +41,11 @@ spec:
         name: manifests
         path: odh-dashboard
     name: odh-dashboard
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: partners/anaconda
+    name: odh-dashboard
```
Comment (cfchase) [inline opendatahub.yaml:48]: I don't think this should have the same name as the odh-dashboard

**Review by cfchase — CHANGES_REQUESTED** [review]

**Review by crobby — CHANGES_REQUESTED** [review]
You're going to need to change the Dockerfile to include the "partners" subdir

**Comment by Xaenalt** [comment]
Added

**Review by crobby — APPROVED** [review]
lgtm

**Review by anishasthana — CHANGES_REQUESTED** [review]
@Xaenalt Can you squash the commits?

**Review by anishasthana — APPROVED** [review]
/lgtm

**Review by cfchase — APPROVED** [review]

**Review by crobby — APPROVED** [review]
I have tested this along with the manifest change and the dashboard change.  I think we're in good shape.

**Pull request merged**